source/content/addons/object-cache/howto/wordpress.md

```diff
@@ -152,7 +150,7 @@ Refer to the [official Object Cache Pro documentation](https://objectcache.pro/d
 	git add auth.json && git commit -m "Add Object Cache Pro auth token."
```

Comment:
It seems odd that we're recommending committing the license to git. That shouldn't be necessary with Secrets Manager.

Reply:
The auth.json is really only for local development/updates. Once we're on Pantheon, the licence is provided by Pantheon environment variables. The convenience of committing it to the repository is that the token is shared between users of the repository (vs everyone having to create their own auth.json), but I've actually tested and confirmed that you don't need auth.json committed to the repository at all. I'm going to walk back this suggestion and add a note that everyone would need to individually pull down/create an auth.json themselves but that it should be gitignored.